### PR TITLE
Support forward compatibility

### PR DESCRIPTION
**Pull request by elad-c**
## Pull Request Description:
Edit the keras quantization wrapper so it is forward compatible with versions 1.4.0 & 1.3.0

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: mct_quantizers/keras/quantize_wrapper.py [hunk 1]
```diff
@@ -199,13 +199,15 @@ def get_config(self):
 
             """
             base_config = super(KerasQuantizationWrapper, self).get_config()
-            config = {WEIGHTS_QUANTIZERS: {k: keras.utils.serialize_keras_object(v) for k, v in self.weights_quantizers.items()},
-                      WEIGHTS_VALUES: {k: self.serialize_fn(v) for k, v in self.weight_values.items()}}
-
+            config = {WEIGHTS_QUANTIZERS: {k: keras.utils.serialize_keras_object(v) for k, v in self.weights_quantizers.items()}}
+            # Only create the wrapper attributes that handle positional weights if they exist, so the wrapper is forward
+            # compatible with older MCTQ versions (at least until MCT with start quantizing positional weights)
```
Comment (ofirgo) [inline mct_quantizers/keras/quantize_wrapper.py:204]: typo (with --> will)
and maybe if it is something that we will need to update once enabling positional weights quantization then make it a todo?

Comment (elad-c) [reply mct_quantizers/keras/quantize_wrapper.py:204]: there's no need to change it when MCT will quantize constants, but it won't be forward compatible for these models anymore.
